frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/NormalizationNotification/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Notification } from "@czi-sds/components";
+import { noop } from "src/common/constants/utils";
+import {
+  StyledIcon,
+  StyledLink,
+  StyledLinkBody,
+  StyledLinkWrapper,
+  StyledNotificationDetails,
+  StyledNotificationLabel,
+  StyledNotificationWrapper,
+} from "../ShareButton/style";
+import { useConnect } from "./connect";
+import {
+  NOMALIZATION_NOTIFICATION_BODY_LINK,
+  NORMALIZATION_NOTIFICATION_BODY,
+  NORMALIZATION_NOTIFICATION_BODY_2,
+  NORMALIZATION_NOTIFICATION_BODY_LINK_LABEL,
+  NORMALIZATION_NOTIFICATION_DOCS_URL,
+  NORMALIZATION_NOTIFICATION_LINK_TEXT,
+  NORMALIZATION_NOTIFICATION_TITLE,
+} from "./constants";
+
+export default function NormalizationNotification(): JSX.Element {
+  const { showURLCopyNotification } = useConnect();
+
+  return (
+    <StyledNotificationWrapper>
+      <Notification
+        key={showURLCopyNotification}
+        autoDismiss={10000}
+        onClose={noop}
+        slideDirection={"left"}
```

Comment:
for string props we can save an expression evaluation by just passing string like so:

```ts
slideDirection="left"
```

here and throughout